frontends/web/src/routes/exchange/pocket.tsx

```diff
@@ -225,15 +223,15 @@ export const Pocket = ({
         const message = serializeMessage({
           version: MessageVersion.V0,
           type: V0MessageType.ExtendedPublicKey,
-          correlationId,
           extendedPublicKey: xpub,
+          correlationId,
         });
         current.contentWindow?.postMessage(message, '*');
       }
     }
   };
 
-  const handleRequestXpub = () => {
+  const handleRequestXpub = (message: RequestExtendedPublicKeyV0Message) => {
```

Comment:
thank you this all makes sense. 

I am thinking of changing this one here to just accept correlationId: string directly.

thank you 🙏  I'll squash your commit into mine and change this to string.